### PR TITLE
Unflag experimental features

### PR DESCRIPTION
**Pull request by natemoo-re**
## Changes

Unflags `--experimental-error-overlay` and `--experimental-prerender`

## Testing

CI

## Docs

N/A

### CODE REVIEW

**Comment by matthewp** [comment]
The test failing here seems really bizarre...

**Comment by natemoo-re** [comment]
> The test failing here seems really bizarre...

This seems to have something to do with the `prerender` changes—I'll investigate.

File: .changeset/chilled-geese-worry.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+'astro': major
+---
+
+The previously experimental features `--experimental-error-overlay` and `--experimental-prerender`, both added in v1.7.0, are now the default.
+
+You'll notice that the error overlay during `astro dev` has a refresh visual design and provides more context for your errors.
```
Comment (delucis) [inline .changeset/chilled-geese-worry.md:7]: Typo:

```suggestion
You'll notice that the error overlay during `astro dev` has a refreshed visual design and provides more context for your errors.
```

File: packages/astro/src/@types/astro.ts [hunk 2]
```diff
@@ -927,46 +925,6 @@ export interface AstroUserConfig {
 	 * These flags are not guaranteed to be stable.
 	 */
 	experimental?: {
-		/**
```
Comment (delucis) [inline packages/astro/src/@types/astro.ts:930]: One thing to note is that removing these will remove them from docs too — https://docs.astro.build/en/reference/configuration-reference/#experimental-flags

Maybe that doesn’t really matter, but we should make sure to say v2-beta required when documenting `prerender` (error overlay’s probably less important docs-wise). I don’t think we got a `prerender` docs PR yet, right @natemoo-re?

Comment (natemoo-re) [reply packages/astro/src/@types/astro.ts:930]: Correct, will let you know when that's ready!

Comment (natemoo-re) [reply packages/astro/src/@types/astro.ts:930]: https://github.com/withastro/docs/pull/2279

**Comment by natemoo-re** [comment]
> The test failing here seems really bizarre...

Figured this out. It was a test that we could customize `vite.build.rollupOptions.output.chunkFileNames`, but the `package.json` didn't include `"type": "module"` and the output files were plain `.js` (which would be treated as CJS in this case). 

Since prerender splits each page out to a separate chunk _instead of a single file_, this broke. Made a specific note in the changelog for this situation.

**Review by matthewp — APPROVED** [review]

**Review by delucis — APPROVED** [review]

**Pull request merged**